internal/cluster/validator.go

```diff
@@ -400,6 +401,26 @@ func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext
 	return ValidationFailure, "The cluster has hosts that are not ready to install."
 }
 
+func (v *clusterValidator) OciManifestsSatisfied(c *clusterPreprocessContext) (ValidationStatus, string) {
+	// If cluster platform type is not OCI ignore that validation
+	if !(c.cluster.Platform != nil && common.PlatformTypeValue(c.cluster.Platform.Type) == models.PlatformTypeOci) {
+		return ValidationSuccess, "OCI platform wasn't set"
```

Comment:
Once you make the validation name generic:
```suggestion
		return ValidationSuccess, "Platform requirements satisfied"
```